src/components/ConversationSettings/BasicInfo.vue

```diff
@@ -53,9 +53,10 @@ import ConversationAvatarEditor from './ConversationAvatarEditor.vue'
 import EditableTextField from '../UIShared/EditableTextField.vue'
 
 import { CONVERSATION } from '../../constants.js'
-import { hasTalkFeature } from '../../services/CapabilitiesManager.ts'
+import { hasTalkFeature, getTalkConfig } from '../../services/CapabilitiesManager.ts'
 
 const supportsAvatar = hasTalkFeature('local', 'avatar')
+const maxDescriptionLength = getTalkConfig('local', 'conversations', 'description-length') || 500
```

Comment:
Should it be not local in future for federated moderators? 🤔

Reply:
it has to take the value of the server hosting the conversation yeah. That being said, I think it might break on the API when a 31 conversation has 2k and invites someone from a 30 server with 500 limit. I'll raise a new issue about this.